config/routes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get 'home/index'
+  resources :messages
+  # get '/messages/random_greeting', to: 'messages#random_greeting'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
```

Comment:
Required
- Kindly remove the namespace api and v1 as you did not version your API because the controller file is directly installed inside the controllers folder, there's no api or v1 directories.